test/test_large_cohort.py

```diff
@@ -122,28 +133,271 @@ def create_config(
+        conf = create_config(
+            tmp_path,
+            seq_type='genome',
+            gnomad_prefix=gnomad_prefix,
+            broad_prefix=broad_prefix,
+        )
+
+        invalid = [
+            'this_shouldnt_work',  # Junk string
+            'CHR1:POS1-CHR1:100M',
+            'CHR2:POS2-CHR1:POS1',  # The start locus ("CHR2:POS2") must precede the end locus ("CHR1:POS1").
+            'CHR1:POS2-CHR1:POS1',  # The start position ("POS2") must be less than the end position ("POS1") within the same contig ("CHR1")
+            'CHR1:POS1-CHR1:START',  # The position "START" stands for 1, so it should not be used as an end position in the interval
+        ]
+
+        context_started = False
+
+        with open(tmp_path / 'intervals.txt'):  # test if can read intervals from txt
+            pass
+
+        for interval in invalid:
+            # Intervals that should break parsing
+            conf.large_cohort['combiner']['intervals'] = [interval]
+
+            # Set config and patch cohort
+            set_config(
+                conf,
+                tmp_path / 'config.toml',
+                merge_with=[DEFAULT_CONFIG, LARGE_COHORT_CONFIG],
+            )
+
+            mocker.patch(
+                'cpg_workflows.inputs.create_cohort',
+                lambda: _mock_cohort(conf.workflow.dataset),
+            )
+
+            # Run the combiner function
+            if not context_started:
+                start_query_context()
+                context_started = True
+
+            res_pref = tmp_path
+            vds_path = res_pref / 'v01.vds'
+            with pytest.raises(FatalError, match='invalid interval'):
+                combiner.run(out_vds_path=vds_path, tmp_prefix=res_pref / 'tmp')
+
+    def test_fails_if_given_duplicate_sequencing_groups(
+        self, mocker: MockFixture, tmp_path: Path
+    ):
+        # Creating a config and modyfing it in some way
+        conf = create_config(
+            tmp_path,
+            seq_type='genome',
+            gnomad_prefix=gnomad_prefix,
+            broad_prefix=broad_prefix,
+        )
+        conf.large_cohort['combiner']['intervals'] = []  # set to empty list
+
+        # Set config and patch cohort
+        set_config(
+            conf,
+            tmp_path / 'config.toml',
+            merge_with=[DEFAULT_CONFIG, LARGE_COHORT_CONFIG],
+        )
+
+        mocker.patch(
+            'cpg_workflows.inputs.create_cohort',
+            lambda: _mock_cohort(conf.workflow.dataset, True),
+        )
+
+        res_pref = tmp_path
+        vds_path = res_pref / 'v01.vds'
+        combiner.run(out_vds_path=vds_path, tmp_prefix=res_pref / 'tmp')
+        print()
+
+    def test_fails_if_given_two_sequencing_groups_with_the_same_gvcf_path(
+        self, mocker: MockFixture, tmp_path: Path
+    ):
+        pass
+
+    @pytest.mark.parametrize('seq_type', ['exome', 'genome'])
+    def test_calls_hail_combiner_with_correct_parameters(
```

Comment:
I would rename this to `test_can_use_default_intervals_if_intervals_are_not_set_in_config`, and delete the tests `test_uses_default_exome_intervals_if_intervals_are_not_specified` and `test_uses_default_genome_intervals_if_intervals_are_not_specified` since they are testing the same thing